utils/rpms/daos.spec

```diff
@@ -328,6 +329,11 @@ mv test.cov-build %{buildroot}/usr/lib/daos/TESTING/ftest/test.cov
 %endif
 mkdir -p %{buildroot}/%{_sysconfdir}/ld.so.conf.d/
 echo "%{_libdir}/daos_srv" > %{buildroot}/%{_sysconfdir}/ld.so.conf.d/daos.conf
+mkdir -p %{buildroot}/%{_sysconfdir}/sysctl.d/
+cat << EOF > %{buildroot}/%{_sysconfdir}/sysctl.d/%{sysctl_script_name}
+net.core.rmem_max=1048576 +
+net.core.rmem_default=1048576
+EOF
```

Comment:
Might be better to have a text file in our code based that can be used here and moved directly by users installing from non-RPM sources, since it looks like both the name and content are static.

Reply:
At the bare minimum I think we need these specific directions in development documentation for when building and running out of the source tree.

Reply:
Is it not sufficient to have the documentation in the admin guide (changes in this PR above)?

Reply:
have made amendments